### PR TITLE
chore: Updated lerna config as per new release process

### PR DESCRIPTION
**Pull request by abhiomkar**
### Description

- Updated `lerna.json` file supported by new lerna version. Set `command.version.conventionalCommits` to `true` which automatically determines the next release version based on our commit conventions.
- Removed CHANGELOG dependencies: `standard-changelog`, `conventional-changelog-angular`, `conventional-commits-parser`, `cz-conventional-changelog` to use lerna provided changelog script.
- Removed check for MDC Web demo site deploy in pre-release script.
- Removed `husky` - commit message validation. Going forward, This commit messages will be validated in PR title using [GitHub bot](https://github.com/probot/semantic-pull-requests)

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,7 +1,13 @@
 {
-  "lerna": "2.0.0-beta.36",
   "version": "2.3.1",
   "command": {
+    "version": {
```
Comment (moog16) [inline lerna.json]: does this automatically run the changelog for us now?

Comment (abhiomkar) [reply lerna.json]: Yes, sending you new release process doc. :)

Comment (moog16) [reply lerna.json]: cool!

File: lerna.json [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+      "allowBranch": [
+        "master",
+        "release/*"
```
Comment (moog16) [inline lerna.json]: why not just use develop here instead of having to create a new release branch?

Comment (abhiomkar) [reply lerna.json]: Removed the need to create `release/*` branch. Updated this config accordingly.

**Comment by moog16** [comment]
ready for rereview?

**Comment by abhiomkar** [comment]
Yes

**Pull request merged**